configuration/defaults/byron-staging/configuration.yaml

```diff
@@ -224,8 +224,6 @@ options:
       - EKGViewBK  
     # Uncomment it to send 'cardano.node.metrics' to 'TraceForwarderBK' as well.
     # - TraceForwarderBK
-    cardano.node-metrics:
-      - EKGViewBK
```

Comment:
Is there some mechanism to make sure these get fixed in the config tempate devops uses?

Reply:
I've made @jbgi aware of this, yes.

Reply:
So this is currently a manual process? Can it not be automated in some way to make sure it always gets done?

Reply:
This template is in a different repository -- `iohk-nix`, and it contains defaults for multiple services.

We've actually been discussing this topic a while ago, but it wasn't feasible then.

cc @disassembler